weekly-q-and-a-sessions/09-12-21.md

```diff
@@ -0,0 +1,54 @@
+# Thursday December 09, 2021
+
+## Attendees
+
+Vegard
+Jakub
+Adi
+Snorre
+Richard
+Benjamin
+Martin
+
+## Notes
+
+### Discussion point: Vegard
+
+Discussion of the breadcrumbs component with regards to https://github.com/fabric-ds/react/pull/66#issuecomment-989643169
+Main issue is that its easy to wrap links in a fragment and assume that this doesnt count (will be stripped out) which is incorrect
+
+### Discussion point: Vegard
+
+Discussion of various features needed and issues encountered when implementing Fabric React components on search results page
+List from Vegard:
+Combobox features:
+- just display options, don’t match
+- don’t catch Enter
+- inputClassName override mb-0
+- listClassName override pb-0
+- disable list above
+- affix icon
+- both prefix and suffix
+- less divs (nested .input classes)
+- affix less divs
+- affix type boolean
+- affix onClick event
```

Comment:
```suggestion
- Add prop for disabling filtering of `options` (show all options, not just those matching `value`). Allows full external control over the `options` list.
- Let Enter `keydown` event bubble up, so that it can be captured by an enclosing `form`. Need in auto-complete to support free-text search when no suggestions are offered.
- Add prop to disable showing the list above the textfield.
- Add support for icons (both svg and img) as `Affix` labels.
- Add support for having both prefix and suffix `Affix` at the same time.
- Remove redundant nested divs from TextField and Affix.
- Extend type of `Affix` child in `TextField`, to support conditional rendering like this:
<TextField value={value}>
    {value && <Affix suffix clear />}
</TextField>
- Extend type of `Affix.toClick()` prop to include `event` parameter (which is actually provided).
```

Reply:
Just tried to explain a bit better.